cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/registry/encrypt/EncodedTicket.java

```diff
@@ -48,7 +49,7 @@ public int getCountOfUses() {
     }
 
     @Override
-    public long getCreationTime() {
+    public ZonedDateTime getCreationTime() {
```

Comment:
Don't think this will work with JPA ticket registries. We'll likely run into either schema or serialization issues. Let's keep this as long.